tests/testthat/test_functions.R

```diff
@@ -0,0 +1,22 @@
+context("Tests of package functions")
+
+mock_data <- data.frame(
+  va = c(1:10),
+  vb = c(2:11),
+  vc = factor(c(rep(1:2, 5)), labels = c("jeden", "dwa")),
```

Comment:
Good practice: let's try to avoid using not English words. Somebody can try to understand your code, especially when it's open.

Reply:
fixed